rpc/cloudagent/network/loadbalancer/moc_cloudagent_loadbalancer.proto

```diff
@@ -39,10 +39,40 @@ message LoadbalancerInboundNatRule {
 	Protocol protocol = 4;
 }
 
+message LoadbalancerOutboundNatRule {
```

Comment:
assuming this is copied from Mittal's changes.
remove this and other non-public ip related proto changes. this is still WIP and we dont want to include it here.